include/common_structs.h

```diff
@@ -1465,8 +1476,8 @@ typedef struct DecorationTable {
     /* 0x8B8 */ s8 decorationType[2];
     /* 0x8BA */ u8 unk_8BA[2];
     /* 0x8BC */ u8 unk_8BC[2];
-    /* 0x8C0 */ s16 unk_8C0[6];
-    /* 0x8BE */ char unk_8BE[30];
+    /* 0x8C0 */ s16 unk_8C0[4];
```

Comment:
I know you didn't introduce this issue, but the field should be called `unk_8BE` (also edit the offset comment please!).